src/db/index.ts

```diff
@@ -53,6 +53,32 @@ export class FrappeDB {
       });
   }
 
+  /**
+   * Get a document from the database
+   * @param {string} doctype Name of the doctype
+   * @param {@type GetDocArgs} [args] Arguments for the query
+   * @returns Promise which resolves to the document object
+   */
+  async getLastDoc<T>(doctype: string, args?: GetLastDocArgs): Promise<FrappeDoc<T>> {
+
+    if (!args || !args?.orderBy) {
+      args = {
```

Comment:
This variable is being self-referenced. Instead, create a new variable called `query_args` like this:

```js
let query_args = {
    orderBy: {
           field: 'creation',
           order: 'desc'
         }
}
```

And then if `args` has been provided in the function call, use this:
```js
if (args){
   query_args = {...query_args, ...args}
}
```

Reply:
What's the error? If it has to do with type, then you should set the type for `query_args` as `GetLastDocArgs`